scripts/provisioning/resetcmd.py

```diff
@@ -116,4 +130,18 @@ def DeleteFileOrDirWithRegex(self, path: str, regex: str):
           sys.stderr.write(f'ERROR: DeleteFileOrDirWithRegex(): Failed to delete: {file}, error: {str(e)}\n')
           raise e
 
+  def purge_messages(self, producer_id: str, msg_type: str, delivery_mechanism: str):
+    """purge messages on message bus."""
+    try:
+      s3MessageBus = S3CortxMsgBus()
+      s3MessageBus.connect()
+      s3MessageBus.setup_producer(producer_id, msg_type, delivery_mechanism)
+      try:
+        s3MessageBus.purge()
+        #Insert a delay of 1 min after purge, so that the messages are deleted
+        time.sleep(60)
```

Comment:
Get purge time from config. same change in bgdelete code as well

Reply:
fixed